modules/dhcp_common/isc/configuration_parser.rb

```diff
@@ -168,12 +168,44 @@ def visit(_, _, _, parent)
         end
         # rubocop:enable Style/StructInheritance
 
+        class Literal < Rsec::Binary
+          def _parse ctx
+            buffer = StringIO.new
+            bs = false
+            end_of_string = false
+            double_quote_counter = 0
+
+            return Rsec::INVALID if ctx.peek(1) != '"'
+            until end_of_string
+              return Rsec::INVALID if ctx.eos?
+              c = ctx.get_byte
+              if bs
+                buffer << sprintf("\\%c", c)
```

Comment:
Can you point me on tests which runs through this branch? I only see regular strings in tests, we might need to add couple more like `'test escape: \\"'` and similar.

Reply:
https://github.com/witlessbird/smart-proxy/blob/a15c8cd8fa6ba218d1c8196ed580ff99a3fa4d47/test/dhcp/conf_parser_test.rb#L439